libs/@local/hash-isomorphic-utils/src/ontology-type-ids.ts

```diff
@@ -0,0 +1,380 @@
+export const systemTypes = {
```

Comment:
I think we should:
1. add `baseEntityTypeUrl` to the types to stop us having to `extractBaseUrl` everywhere (probably worth doing for entity types as well as property types)
2. Split this into different `systemEntityTypes`, `systemPropertyTypes` objects so that importers are dealing with smaller objects in memory if they don't need it all, as well as helping for bundling / tree shaking for contexts where bundle size is an issue

Reply:
I've addressed `1` in https://github.com/hashintel/hash/pull/3551/commits/24ca71b38a286f2fb347db100a443ef1783d1f33, and `2` in https://github.com/hashintel/hash/pull/3551/commits/41b2c48a8fcd14ed1c052ddccb9fefadbfdc7cad